saleor/plugins/avatax/plugin.py

```diff
@@ -136,6 +138,16 @@ class AvataxPlugin(BasePlugin):
             ),
             "label": "Shipping tax code",
         },
+        "override_global_tax": {
+            "type": ConfigurationTypeField.BOOLEAN,
+            "help_text": "Used when setting per channel is needed.",
```

Comment:
Maybe something that will tell us that it overwrites global settings for include taxes in prices. `Overwride global tax` can be a little bit too generic.

Reply:
Yep I had the same feeling. At least we could give an example which options would be overridden.

Reply:
Can we make it a radio group or a drop-down with three options:

```
Prices:
   (o) Respect Saleor's configuration (default)
   ( ) Treat all prices as net amounts (excl. taxes)
   ( ) Treat all prices as gross amounts (inc. taxes)
```

Reply:
@patrys sure, you described `label` what I should put in the `help_text`?

Reply:
I started to set this up, and I recall there is no way of getting choices input (radio or select) in the plugin settings. I discussed this with @korycins, and we agreed that two checkboxes solution is good enough.

@patrys, do you still want to change labels with this additional context? If yes, please provide me with labels that suites you.